pkg/wpa/passphrase/passphrase_test.go

```diff
@@ -0,0 +1,101 @@
+package passphrase
+
+import (
+	"bytes"
+	"testing"
+)
+
+type RunTestCase struct {
+	name      string
+	essid     string
+	pass      string
+	out       []byte
+	errExists bool
+}
+
+var (
+	essidStub     = "stub"
+	shortPass     = "aaaaaaa"                                                          // 7 chars
+	longPass      = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" // 64 chars
+	validPass     = "aaaaaaaaaaaaaaaa"                                                 // 16 chars
+	correctOutput = []byte(
+		`network={
+	ssid="stub"
+	#psk="aaaaaaaaaaaaaaaa"
+	psk=e270ba95a72c6d922e902f65dfa23315f7ba43b69debc75167254acd778f2fe9
+}
+`) // indentation matters
+
+	runTestCases = []RunTestCase{
+		{
+			name:      "No essid",
+			essid:     "",
+			pass:      validPass,
+			out:       nil,
+			errExists: true,
```

Comment:
make this a string with fmt.Errorf. Then compare that.

Reply:
Fixed